src/tree/tree.scss

```diff
@@ -17,6 +17,28 @@ $iui-expander-button-width: ($iui-icons-default) + ($iui-expander-inline-padding
   list-style: none;
 }
 
+@mixin iui-tree-item {
+  outline: none;
```

Comment:
This one removes outline from whole sub-tree.

Reply:
Can you move it inside focus-visible?

Reply:
Why?

Reply:
To avoid removing default outline in older browsers where our focus-visible mixin doesn't work

Reply:
So it will be even worse, because it will add focus on node and its sub-tree :/

Reply:
How is it worse than no focus ring?

Reply:
Idk, it is sounds bad in both ways 😀 What is the purpose of this mixin if it isn't even working on older browsers?

Reply:
It works in most browsers and helps avoid code duplication and focus bugs when comparing with the other approach (`focus:not(:focus-visible)`). I don't understand why you want to show no outline rather than the default.

Reply:
"most browsers" is latest ones? So it is worthless to me as it should make `:focus` work but it doesn't... But whatever, moved.